### PR TITLE
Adding translation to portuguese

### PR DESCRIPTION
**Pull request by mailysantos**
## Description

This PR has requested changes in the issue regarding the translation into Portuguese of the texts contained in the file translations.yml.

Issue #98 - Add Portuguese translations for general strings 

## Required checklist:

* [x] My code matches the existing code style of this project.
* **One** of these:
	* [ ] This is a new localization for existing content.
	* [ ] This is new content that is localized for all the locales that this project currently supports.
	* [x] I described above how I plan to localize this content immediately after merging these changes.
	* [ ] Localization is not applicable to this change.
* [x] I have tested my code and it successfully builds for GitHub Pages and validates as AMP.

### CODE REVIEW

**Comment by devadvance** [comment]
Hey @mailysantos, great work! To resolve the merge conflicts, I recommend moving the `id` (Indonesian) translations to be above your Portuguese translations. That should resolve them.

For example, changing this:

```
  pt:
    name: Português
    prefix: /pt
  id:
    name: Bahasa
    prefix: /id
```

to this:
```
  id:
    name: Bahasa
    prefix: /id
  pt:
    name: Português
    prefix: /pt
```

File: _data/translations.yml [hunk 1]
```diff
@@ -31,6 +34,8 @@ site_description:
     के लिए आपको आवश्यक सभी ज्ञान।
   fr: Toutes ce que vous avez besoin de savoir pour
     utiliser des terminaux sur différents systèmes d'exploitation.
+  pt: Todo o conhecimento de que você precisa para começar a trabalhar
```
Comment (devadvance) [inline _data/translations.yml:37]: Removing the extra space at the end of this line will fix the automated check, and then we can approve and merge.

Comment (mailysantos) [reply _data/translations.yml]: Hey, I remove the extra space. Thanks for the tips!

**Review by devadvance — APPROVED** [review]
Looks good to me. @mailysantos fantastic contribution! Muito obrigado!

**Comment by mailysantos** [comment]
Thank you, @devadvance! It is a very interesting project, I'll continue to follow. See you

**Pull request merged**